test/e2e/specs/specs-playwright/wp-editor__navbar.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * @group gutenberg
```

Comment:
I think we should run this on `calypso-pr` runs, right? That navbar is pure calypso, and we override some stuff in the nav sidebar, so I think it's directly relevant to PR runs!

Reply:
Oh, I completely forgot about this one - I copy-pasted the test contents and simply forgot to update it.